.github/workflows/StaleIssues.yml

```diff
@@ -0,0 +1,20 @@
+name: Mark stale issues
+
+on:
+  schedule:
+  - cron: '44 5 * * 2' # Tuesday, 5:44
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v9.1.0
+      with:
+        stale-issue-label: 'stale'
+        days-before-issue-stale: 180
+        days-before-issue-close: 14
+        close-issue-message: 'There has been no recent activity on this issue. Please re-open or create a new issue, if needed.'
```

Comment:
Can we add value for the `stale-issue-message` parameter as well? Something like

"There has been no recent activity on this issue. If this is still an issue, please add a new comment."

Reply:
I intentionally omitted that part as I don't think it's necessary.
If the issue doesn't need to be closed, they can still comment, and we'll reopen it. Worst case, new issue is created.
It's a bit more aggressive approach, I know.